File: src/main/java/com/github/pnemonic/game/lottery/pais777/Lotto777SubStats.java
package com.github.pnemonic.game.lottery.pais777;

import com.github.pnemonic.game.lottery.Lottery;

import java.io.File;

public class Lotto777SubStats extends Lotto777Stats {

	public Lotto777SubStats(Lottery lottery) {
		super(lottery);
	}

	/**
	 * @param args
	 */
	public static void main(String[] args) {
		String fileName = (args.length == 0) ? "results/777.csv" : args[0];
		File file = new File(fileName);
		Lotto777Stats stats = new Lotto777Stats(new Lotto777Sub());
		stats.parse(file);
	}

}

File: src/main/java/com/github/pnemonic/game/lottery/pais777/Lotto777SubStats.kt
package com.github.pnemonic.game.lottery.pais777

import java.io.File

class Lotto777SubStats {
    companion object {
        fun main(args: Array<String>) {
            val fileName = if (args.isEmpty()) "results/777.csv" else args[0]
            val file = File(fileName)
            val stats = Lotto777Stats(Lotto777Sub())
            stats.parse(file)
        }
    }
}